book/publishing/about.md

```diff
@@ -0,0 +1,49 @@
+(pb-about)=
+
+# About EDS book notebooks
+With the great diversity in open data, software and computational resources, environmental data science research and applications have accelerated rapidly. 
+This presents a timely opportunity for advancing the cyberinfrastructure for showcasing these emerging developments. 
+Building upon existing open science communities such as the [Turing Way](https://the-turing-way.netlify.app/) and [Pangeo](https://pangeo.io), EDS book aims at educating and leveraging good scientific software and data management practises among environmental scientists through peer-reviewed findable, accessible, interoperable and reusable (FAIR) computational notebooks. 
+
+Computation notebooks (CNs) allow users to combine interactive code with text and graphical objects. 
+CNs have become a successful mechanism for sharing the analysis of data and documenting research workflows. 
+Notebooks has been an excellent tool for prototyping and writing examples to showcase a piece of software. 
+
+The EDS book aims to complement the science and methodological development embedded within academic journals using open infrastructure to translate research outputs into FAIR notebooks which benefit from a collaborative and open reviewing process (Figure 1).
```

Comment:
Figure 1 is referred to as Figure 2 under the image itself

Reply:
Fixed in [00db7fa](https://github.com/alan-turing-institute/environmental-ds-book/pull/117/commits/00db7fa2dd4529f715085ba71b306f68c3859a83).